express/parsers/apps/espresso/formats/txt.py

```diff
@@ -275,19 +278,43 @@ def initial_lattice_vectors(self, text):
                     'a': [-0.561154473, -0.000000000, 0.561154473],
                     'b': [-0.000000000, 0.561154473, 0.561154473],
                     'c': [-0.561154473, 0.561154473, 0.000000000],
-                    'alat': 9.44858082
+                    'alat': 1
                 }
             }
         """
+        lattice_alat = self._lattice_alat(text)
```

Comment:
1. Rename `self._lattice_alat` => `self._get_alat` in the class
2. `lattice_alat` => `alat` in the function
3. On line 286:
`lattice_in_alat_units` = self._extract_lattice(text, regex="lattice_alat")